src/sparseml/transformers/__init__.py

```diff
@@ -40,11 +40,10 @@ def _check_transformers_install():
     # check for NM integration in transformers version
     import transformers as _transformers
 
-    if not _transformers.NM_INTEGRATED:
+    if not getattr(_transformers, "NM_INTEGRATED", False):
         _LOGGER.warning(
-            "the neuralmagic fork of transformers may not be installed. it can be "
-            "installed via "
-            f"`pip install {_NM_TRANSFORMERS_NIGHTLY}`"
+            "The neuralmagic fork of transformers may not be installed. It can be "
+            "installed via `pip install nm_transformers`"
```

Comment:
nit: the PyPI package name is `nm-transformers` (hyphen, not underscore). Using an underscore instead will work with `pip install` though I think matching the name more accurately is better for consistency/etc.